pkgs/activitywatch/default.nix

```diff
@@ -185,33 +178,37 @@ rec {
 
     nativeBuildInputs = [
       python3.pkgs.poetry
-      python3.pkgs.pyqt5 # for pyrcc5
-      libsForQt5.wrapQtAppsHook
-      xdg-utils
+      qt6.wrapQtAppsHook
     ];
 
     propagatedBuildInputs = with python3.pkgs; [
       aw-core
-      pyqt5
+      qt6.qtbase # Needed for qt6.wrapQtAppsHook
+      qt6.qtsvg # Rendering icons in the trayicon menu
```

Comment:
It seems that without this library, [this code](https://github.com/ActivityWatch/aw-qt/blob/8ec5db941ede0923bfe26631acf241a4a5353108/aw_qt/trayicon.py#L124-L130) detects the missing library as an error and doesn't render the icon. Should we report this upstream as possibly a required dependency to fix the issue mentioned in the comment?

Reply:
Interesting, perhaps report it to https://github.com/ActivityWatch/activitywatch/issues/62.

Reply:
Reported upstream :+1: